distribution/docs/src/test/java/ddf/docs/DocumentationTest.java

```diff
@@ -46,11 +44,11 @@
 
   private static final String BASE64_MISSING = "base64,\"";
 
-  private static final String HREF_ANCHOR = "href=\"#_";
+  private static final String HREF_ANCHOR = "href=\"#";
```

Comment:
It might be useful to explain what each of the elements means in the context of the documentation, including these anchors, the underscores and the colon references.